demo/index.html

```diff
@@ -113,11 +113,19 @@
     <li><a href="#" class="wra">Friedan, Betty</a></li>
   </ul>
 
+  <ul id="select-root" tabindex="0">
+    <li class="select">Number 1</li>
+    <li class="select">Number 2</li>
+    <li class="select">Number 3</li>
+    <li class="select">Number 4</li>
+  </ul>
+
   <p>
     <span style="font-size:2em;vertical-align:middle;">☜</span>
     <a href="https://github.com/davidtheclark/focus-group" style="vertical-align:middle;">Return to the repository</a>
   </p>
 
+  <script src="https://unpkg.com/mitt/dist/mitt.umd.js"></script>
```

Comment:
Totally 👍  wasn't sure how you wanted to handle it so I just threw it in the html.